dealership/contracts.py

```diff
@@ -1,12 +1,46 @@
-class Contract(object):
-    pass
+from dealership.vehicles import Car, Truck, Motorcycle
+from dealership.customers import Customer, Employee
 
+class Contract(object):
+    
+    def __init__(self, vehicle, customer):
+        self.vehicle = vehicle
+        self.customer = customer
+        self.discount = 0
+        if self.customer.is_employee():
+            self.discount = 0.1
+        
+    def total_value(self):
+        raise NotImplementedError()
+        
+    def monthly_value(self):
+        raise NotImplementedError()
+        
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
-
+        super(BuyContract, self).__init__(vehicle, customer)
+        self.monthly_payments = monthly_payments
+        
+    def total_value(self):
+            
```

Comment:
You've got some weird spacing choices (just like me!) inside your methods. You could probably clean those up a bit to improve readability.

Very nice job, Chee Yim!

Reply:
Got it! Learnt that I need to add backslash to write codes in multiple lines (if we want to break up lines that are too long). Also clean up the spaces.. 

Thanks for the constructive feedback @brianr001 !